google-api-go-generator/gen.go

```diff
@@ -741,6 +749,9 @@ func (a *API) GenerateCode() ([]byte, error) {
 		pn("opts = append([]option.ClientOption{scopesOption}, opts...)")
 	}
 	pn("opts = append(opts, internaloption.WithDefaultEndpoint(basePath))")
+	if a.mtlsAPIBaseURL() != "" {
+		pn("opts = append(opts, internaloption.WithDefaultMTLSEndpoint(mtlsBasePath))")
+	}
```

Comment:
keep here and L729 consistent. people *do* sometimes edit this code by hand.

suggest adding the same if statement above or change this to have the if statement in the generated code.

like this suggestion

```suggestion
pn("if mtlsBasePath != "" { opts = append(opts, internaloption.WithDefaultMTLSEndpoint(mtlsBasePath)) }")
```